[2022-08-17 15:30:57,079] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: my_dag.accurate manual__2022-08-17T15:30:53.369419+00:00 [queued]>
[2022-08-17 15:30:57,094] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: my_dag.accurate manual__2022-08-17T15:30:53.369419+00:00 [queued]>
[2022-08-17 15:30:57,095] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-17 15:30:57,096] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-17 15:30:57,096] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-17 15:30:57,112] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): accurate> on 2022-08-17 15:30:53.369419+00:00
[2022-08-17 15:30:57,118] {standard_task_runner.py:52} INFO - Started process 2289 to run task
[2022-08-17 15:30:57,122] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'accurate', 'manual__2022-08-17T15:30:53.369419+00:00', '--job-id', '64', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmp82l1ynjl', '--error-file', '/tmp/tmpubjpw178']
[2022-08-17 15:30:57,124] {standard_task_runner.py:80} INFO - Job 64: Subtask accurate
[2022-08-17 15:30:57,212] {task_command.py:371} INFO - Running <TaskInstance: my_dag.accurate manual__2022-08-17T15:30:53.369419+00:00 [running]> on host 9d5d2a77e6ba
[2022-08-17 15:30:57,336] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=accurate
AIRFLOW_CTX_EXECUTION_DATE=2022-08-17T15:30:53.369419+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-17T15:30:53.369419+00:00
[2022-08-17 15:30:57,342] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-17 15:30:57,345] {subprocess.py:74} INFO - Running command: ['bash', '-c', "echo 'accurate'"]
[2022-08-17 15:30:57,367] {subprocess.py:85} INFO - Output:
[2022-08-17 15:30:57,373] {subprocess.py:92} INFO - accurate
[2022-08-17 15:30:57,374] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-17 15:30:57,436] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=accurate, execution_date=20220817T153053, start_date=20220817T153057, end_date=20220817T153057
[2022-08-17 15:30:57,497] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-17 15:30:57,539] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
